### PR TITLE
Bitcoin Core 0.13.0

### PR DESCRIPTION
**Pull request by laanwj**
Bitcoin Core 0.13.0 release announcement.

Can be merged if travis passes.

### CODE REVIEW

**Comment by kanzure** [comment]
https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-August/013055.html

**Comment by laanwj** [comment]
Looks like it still expects the old names:

**Comment by laanwj** [comment]
I fixed most of the html errors, now it complains about a unterminated `</a>`
Can anyone get access to the generated html to see what is going wrong here? @Cobra-Bitcoin maybe?
```
- ./_site/en/release/v0.13.0.html
  *  Unexpected end tag : a
```

File: _releases/v0.13.0.md [hunk 1]
```diff
@@ -0,0 +1,894 @@
...
+
+
+Low-level ZMQ changes
+----------------------
+
+- Each ZMQ notification now contains an up-counting sequence number that allows
+  listeners to detect lost notifications.
+  The sequence number is always the last element in a multi-part ZMQ notification and
+  therefore backward compatible. Each message type has its own counter.
+  PR [#7762](https://github.com/bitcoin/bitcoin/pull/7762).
```
Comment (MarcoFalke) [inline _releases/v0.13.0.md]: Try to remove the `#`?

**Pull request merged**